07.ls_object/ls_file.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class LsFile
+  PERMISSION = {
+    '0' => '---',
+    '1' => '--x',
+    '2' => '-w-',
+    '3' => '-wx',
+    '4' => 'r--',
+    '5' => 'r-x',
+    '6' => 'rw-',
+    '7' => 'rwx'
+  }.freeze
+
+  attr_reader :file_name
+
+  def initialize(directory_path:, file_name:)
```

Comment:
LsDirectoryと同じく、キーワード引数にする必要はないと思います。
（まずは普通の引数でメソッド定義するようにして、何かしっくりこない部分があればキーワード引数にすることを検討する）

```suggestion
  def initialize(directory_path, file_name)
```